index.js

```diff
@@ -0,0 +1,34 @@
+const API_KEY = `3265874a2c77ae4a04bb96236a642d2f`;
+const form = document.querySelector("form");
+const search = document.querySelector("#search");
+const weather = document.querySelector("#weather");
+
+const getWeather = async (city) => {
+  weather.innerHTML = `<h2> Loading... <h2>`;
```

Comment:
you can create a separate function as `showLoader()` and move `weather.innerHTML = `<h2> Loading... <h2>`;` to that function.